projects/ad_fmclidar1_ebz/README.md

```diff
@@ -0,0 +1,103 @@
+
+# AD_FMCLIDAR1_EBZ HDL reference design
+
+## Overview
+
+The following design supports both Xilinx and Intel FPGA's. The [AD_FMCLIDAR1_EBZ](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/AD-FMCLIDAR1-EBZ.html)
+prototyping system connects to the FPGA carrier board through a FMC (FPGA Mezzanine Cad)
+high pin count connector.  
+
+Currently supported carriers:
+
+|  Carrier name | FMC connector |
+| ------------- | ------------- |
+|  ZC706        |   FMC_HPC     |
+|  ZCU102       |   HPC0        |
+|  Arria10SOC   |   FMCA_HPC    |
+
+The design is easily portable to any Xilinx or Intel FPGA carrier board, which
+has an FMC HPC connector, and have all the required connections. (See more info
+at IO layout)
+You can find a porting guide in the [wiki.analog.com](https://wiki.analog.com/resources/fpga/docs/hdl/porting_project_quick_start_guide).
+
+### Directory Structure
```

Comment:
The directory structure doesn't look good in the browser, may need additional formatting

Reply:
Fixed. Thanks!